lib/kc_elixir_web/controllers/page_controller.ex

```diff
@@ -1,7 +1,49 @@
 defmodule KcElixirWeb.PageController do
   use KcElixirWeb, :controller
 
+  @meetups [
+    %{
+      datetime: [
+        ~N{2019-03-07 18:00:00},
+        ~N{2019-03-07 20:00:00}
+      ],
+      image_url: "http://placekitten.com/200/200",
+      topic: "Dialyzer!",
+      location: %{
+        url: "http://www.403club.com",
+        address: "614 Reynolds Ave, Kansas City, KS 66101",
+        venue: "403 Club"
+      },
+      description: """
```

Comment:
Also lets use this description:

https://github.com/kcelixir/schedule/pull/3/files